src/HamiltonianBuilder/DOCI.cpp

```diff
@@ -109,42 +147,83 @@ Eigen::VectorXd DOCI::matrixVectorProduct(const HamiltonianParameters& hamiltoni
     size_t dim = this->fock_space.get_dimension();
     // Create the first spin string. Since in DOCI, alpha == beta, we can just treat them as one
```

Comment:
```suggestion
    // Create the first spin string. Since in DOCI, alpha == beta, we can just treat them as one and multiply all contributions by 2
```